tests/setup/globalDefines.ts

```diff
@@ -1,5 +1,5 @@
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const g = global as any
+const g = globalThis as any
```

Comment:
Unfortunately, this was the only way I could find to get something in the global scope. Using config or exports simply wouldn't work. Same counts for the global testHelpers.